src/popup/router/components/Modals/SignModal.scss

```
@@ -6,6 +6,7 @@
   overflow-y: scroll;
```

won't changing this to
```suggestion
  overflow-y: auto; // or even removing this
```
will work the same?

---

Yes, you are right